### PR TITLE
Fix/pyjwt pyquil

### PR DESCRIPTION
**Pull request by cqc-melf**
removes the temporary solution for https://github.com/CQCL/pytket/issues/153

### CODE REVIEW

**Comment by cqc-melf** [comment]
I have added the execution of the two ci checks on all PRs, is this something we want to keep? I would be fine with having that run on every PR.

**Comment by cqc-melf** [comment]
I have added a check of the requirements file to the conditions of running the checks, I think this is the better solution

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+    pytket == 1.4.3
+    pytket-pyquil == 0.25.0
```
Comment (cqc-alec) [inline requirements.txt:2]: Put in alphabetical order?

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (cqc-melf) [reply requirements.txt]: Yes, done. Thank you.

**Pull request merged**